### ISSUE TITLE
[4.0] intro image php notice on blog on frontend

### ISSUE THREAD

**Issue by PhilETaylor** [joomla/joomla-cms#20490]
### Steps to reproduce the issue

install 4.0-dev at 79fd949
install sample data
login to frontend as super admin
click blog

### Expected result

blog view no php notices

### Actual result

```
Notice
: Undefined property: stdClass::$image_intro_caption in
/Users/phil/Sites/joomla-cms/layouts/joomla/content/intro_image.php
on line
16
```
<img width="918" alt="screen shot 2018-05-19 at 22 02 34" src="https://user-images.githubusercontent.com/400092/40273018-642e456e-5bb0-11e8-94ff-e5b013ab22b9.png">

### System information (as much as possible)
79fd94942e1758cff32da7f8f368b5f78cc75c40

**Comment by laoneo** [comment]
Will this pr #20503 fix the issue?

**Comment by Quy** [comment]
Fixed in PR #20503<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20490">issues.joomla.org/tracker/joomla-cms/20490</a>.</sub>
